digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/MunicipalityApi.scala

```diff
@@ -220,7 +220,7 @@ class MunicipalityApi(val onOffLinearAssetService: OnOffLinearAssetService,
   }
 
   def updateSpeedLimitAsset(assetId: Long, parsedBody: JValue, linkId: Long): (SpeedLimit, RoadLink) = {
-    val oldAsset = speedLimitService.getSpeedLimitAssetsByIds(Set(assetId)).filterNot(_.expired).headOption
+    val oldAsset = speedLimitService.getSpeedLimitAssetsByIds(Seq(assetId)).filterNot(_.expired).headOption
```

Comment:
The seq of assetId should be only with unique ids, why this method was changed to Seq instead of Set?